.golangci.yml

```diff
@@ -14,9 +14,7 @@ linters:
     - forbidigo
     - gochecknoinits
     - goconst
-    - gofmt
```

Comment:
would be nice to know why these linters are disabled

Reply:
reverted